src/resources/app/pages/courseStream.vue

```diff
@@ -1,15 +1,79 @@
 <template>
-  <v-card elevation="2" outlined shaped>
-    <v-card-text>
-      <h2 id="code-title">Class code</h2>
-      <div id="class-code">R00M00</div>
-      <v-btn @click="copyCode('R00M00')" id="copy-btn"> Copy </v-btn>
-    </v-card-text>
-  </v-card>
+  <v-container>
+    <v-row>
+      <v-col md="12" class="mb-4">
+        <v-card class="rounded-lg" elevation="0" color="primary">
+          <v-card-text id="course-title"> Computer Networks </v-card-text>
+        </v-card>
+      </v-col>
+    </v-row>
+    <v-row>
+      <v-spacer></v-spacer>
+      <v-col
+        v-for="tab in tabs"
+        sm="2"
+        :key="tab.name"
+        class="flex justify-center"
+      >
+        <v-btn
+          class="mt-1"
+          depressed
+          :color="activeTab == tab.name ? 'light' : 'white'"
+          @click="redirectTab(tab.href)"
+        >
+          {{ tab.name }}
+        </v-btn>
+      </v-col>
+      <v-spacer></v-spacer>
+    </v-row>
+    <v-row>
+      <v-col v-if="activeTab == 'Main Page'" md="2" class="mt-4">
+        <v-card outlined class="rounded-lg">
+          <v-card-text>
+            <h2 id="code-title">Class code</h2>
+            <div class="font-bold text-lg" id="class-code">R00M00</div>
+            <v-btn
+              @click="copyCode('R00M00')"
+              id="copy-btn"
+              class="mt-1"
+              elevation="0"
+              outlined
+            >
+              Copy
+            </v-btn>
+          </v-card-text>
+        </v-card>
+      </v-col>
+      <v-col md="10">
+        <slot />
+      </v-col>
+    </v-row>
+  </v-container>
 </template>
 
 <script>
+import AppLayout from "../components/Layout/AppLayout.vue";
 export default {
+  layout: [AppLayout],
+  data() {
+    return {
+      tabs: [
+        {
+          name: "Classwork",
+          href: "/classwork",
+        },
+        {
+          name: "People",
+          href: "/people",
+        },
+        {
+          name: "Main Page",
```

Comment:
If you don't mind, please change `Main Page` to `Announcements` (don't forget to use plural form), and please change the order to :
1. Announcements
2. Classwork
3. People

Reply:
okay, i was referencing from figma earlier